src/detect.py

```diff
@@ -60,104 +51,30 @@ def image_callback(self, image):
         # Store value on a private attribute
         self._current_image = image
 
-    def pc_callback(self, pc):
-        """Point cloud callback"""
-        # Store value on a private attribute
-        self._current_pc = pc
-
     def run(self):
         # run while ROS runs
         while not rospy.is_shutdown():
             # only run if there's an image present
             if self._current_image is not None:
                 try:
-                    # if the user passes a fixed frame, we'll ask for transformation
-                    # vectors from the camera link to the fixed frame
-                    # if self._global_frame is not None:
-                    #     (trans, _) = self._tf_listener.lookupTransform('/' + self._global_frame,
-                    #                                                    '/' + self._frame, rospy.Time(0))
+                    #Search image
                     small_frame = self._bridge.imgmsg_to_cv2(self._current_image, desired_encoding='bgr8')
-                    # small_frame = self._bridge.imgmsg_to_cv2(, 'rgb8')
+                    
+                    #Load model
                     yolo = YOLO("yolov8n.pt")
                     results = yolo.predict(source=small_frame, conf=0.8)
                     boxes = results[0].boxes
+                    
+                    #Plot bbox 
                     small_frame = pr.plot_bboxes(small_frame, boxes.boxes, conf=0.5)
-
-                    #marked_image = np.squeeze(boxes)
-                    for r in results:
```

Comment:
Should this be removed?

Reply:
Yes, was used when we were ploting bbox with cv2.